docs/submit-opentargets-batch.md

```diff
@@ -1,25 +1,44 @@
+At the end of each step there is a list of checks do be done during review of the ticket.
+
+Log in to the LSF cluster, where all data processing must take place. You must use a common EVA production user instead of your personal account. Follow the [Build instructions](build.md). In particular, you'll need to install Python 3.5 (if you don't have it already), build the Java ClinVar parser, build and install the Python pipeline.
 
 ## Set up environment
-Commands below depend on a number of environment variables. It makes sense to set them all at once before executing any of the steps.
+Commands throughout the protocol depend on a number of environment variables. It makes sense to set them all at once before executing any of the steps. Carefully check the validity of all values. Some of them are announced in the e-mail which they send a few weeks before the data submission deadline:
+* Year and month of the upcoming Open Targets release (`OT_RELEASE`). For example, if you're processing data for “20.02” release, this variable will be set to `2020-02`.
+* Open Targets JSON schema version (`OT_SCHEMA_VERSION`)
+* Open Targets validator package version (`OT_VALIDATOR_VERSION`)
+
+Next you will need to determine which ClinVar release to use as source (`CLINVAR_RELEASE`). Each Open Targets release is synchronised with a certain Ensembl release version, which is also announced in the aforementioned e-mail. Each Ensembl release is, in turn, synchronised with a certain ClinVar version. Based on Ensembl version, we can find the ClinVar release associated to an Ensembl release in its [sources page](http://www.ensembl.org/info/genome/variation/species/sources_documentation.html). For example, if Ensembl is using ClinVar version “07/2019”, this variable will be set to `2019-07`. Note that this is generally *different* from the Open Targets release year and month.
 
 ```bash
-# Year and month for the upcoming OpenTargets release. This is announced in their e-mail.
+# Year and month for the upcoming Open Targets release
 export OT_RELEASE=YYYY-MM
 
-# Year and month of ClinVar release used (see step 1.1 Download ClinVar data).
-# Note that this is *different* from the OpenTargets release year/month.
+# Year and month of ClinVar release used
 export CLINVAR_RELEASE=YYYY-MM
 
-# This variable should point to the directory where this repository clone is located on the cluster.
+# Open Targets JSON schema version
+export OT_SCHEMA_VERSION=1.6.3
+
+# Open Targets validator schema version
+export OT_VALIDATOR_VERSION=0.5.0
+
+# This variable should point to the directory where the clone of this repository is located on the cluster
 export CODE_ROOT=/nfs/production3/eva/software/eva-cttv-pipeline
 
-# Setting up Python version
+# Setting up Python version (the same one which you installed using build instructions)
 PYTHON_VERSION=3.5.6
 INSTALL_PATH=/nfs/production3/eva/software/python-${PYTHON_VERSION}
```

Comment:
Sorry to be a pain here but I am really antsy about having production file system paths in a public repo. Is it possible to work around this at all?....

Reply:
Agreed. I removed production paths: https://github.com/EBIvariation/eva-cttv-pipeline/pull/87/commits/f57cb098d3986c6a516c7df766072b1b21b6805a and added them into the private "configuration" repository